backend/src/v5/routes/teamspaces/teamspaces.js

```diff
@@ -191,10 +191,13 @@ const establishRoutes = () => {
 	 *                         type: string
 	 *                         description: Name of the company
 	 *                         example: 3D Repo Ltd
-	 *                       job:
-	 *                         type: string
-	 *                         description: Job within the teamspace
-	 *                         example: Project Manager
+	 *                       roles:
+	 *                         type: array
+	 *                         items:
+	 *                           type: string
+	 *                           format: uuid
```

Comment:
Since I don't see a change here or an addition of middleware here, I'm guessing  are we converting the UUIDs to string here?

Can we sanity check all the routes are converting UUIDs to string, and the database is storing UUIDs?